src/utils/material.js

```diff
@@ -127,3 +131,9 @@ function handleTextureEvents (el, texture) {
   });
 }
 module.exports.handleTextureEvents = handleTextureEvents;
+
+module.exports.isHLS = function (videoEl) {
+  if (videoEl.type && videoEl.type.toLowerCase() in ['application/x-mpegurl', 'application/vnd.apple.mpegurl']) { return true; }
```

Comment:
it doesn't work for me in Chrome nor iOS - you sure?

![screenshot of Chrome](https://cloud.githubusercontent.com/assets/203725/26021684/e7a47b04-3747-11e7-85f6-82377da26d5e.png)

[here's a test case](http://jsbin.com/qakosuhude) I tested on iOS.